### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by DDDaar** [FYYDCC/IVT-LR#3]
Hello, I noticed the line train_dataset = dataset["train"].select(range(20)).filter(has_image) in the code. Were the results reported in the paper obtained by training on only 20 samples, or should this be modified to use the entire dataset? Additionally, I would like to ask for an estimate of the total training time.

**Comment by FYYDCC** [comment]
Thank you for interest. 
The select(range(20)) is for debugging purposes.
In terms of training time, it takes roughly 12 hours to train qwen2-VL. For Chameleon, the training time is longer, approximately 24 hours.

**Comment by DDDaar** [comment]
> Thank you for interest. The select(range(20)) is for debugging purposes. In terms of training time, it takes roughly 12 hours to train qwen2-VL. For Chameleon, the training time is longer, approximately 24 hours.

Thanks! Another question: for the no-cot method, are you using the original Qwen2-VL-7B-Instruct model that hasn't been trained on the M3CoT or SQA datasets? Or was it actually trained on those specific datasets（sft on answer token）?

**Comment by FYYDCC** [comment]
The accuracy reported in the paper for the no-CoT setting is based on the original model without any additional training. 
We also experimented with an explicit text-only SFT setup. However, the performance of that variant was still significantly lower than our proposed method.

**Comment by DDDaar** [comment]
> The accuracy reported in the paper for the no-CoT setting is based on the original model without any additional training. We also experimented with an explicit text-only SFT setup. However, the performance of that variant was still significantly lower than our proposed method.

Thanks for the reply. May I ask which evaluation method or specific paper the no-CoT version results were derived from? I used VLMEvalKit to evaluate the Qwen2-VL-7B model on the ScienceQA dataset, and my results seem to be higher than the no-CoT figures provided here

**Comment by FYYDCC** [comment]
The no-CoT results reported in our paper are obtained using a pure 0-shot evaluation with the original model. We directly prompt the model and use a simple regular expression to extract the final answer option from the generated output.

Our evaluation follows the same evaluation logic as the inference script provided in our repository, and we did not use any external evaluation framework.
